.gitignore

```diff
@@ -4,3 +4,4 @@
 coverage/*
 notes.md
 tests/fixtures/cassette*
+composer.lock
```

Comment:
The advantage for composer.lock in my opinion is regarding the library contributors, it creates a safe setup for them start coding on it, what do you think?